docs/source/user_guide/installation/install-from-docker.rst

```diff
@@ -26,12 +26,6 @@ Use the following command to pull the PyBaMM Docker image from Docker Hub:
 
             docker pull pybamm/pybamm:latest
 
-.. tab:: Scikits.odes solver
-
-      .. code:: bash
-
-            docker pull pybamm/pybamm:odes
-
```

Comment:
`scripts/Dockerfile` needs to be updated too to remove the `ODES` arg

Reply:
Done already